### ISSUE TITLE
Better documentation

### ISSUE THREAD

**Issue by arashsa** [attardi/deepnl#57]
It would be great if there was more documentation/code that showed how to use the various functions, and how to train models.

**Comment by samoliver360** [comment]
I agree totally with you. It's very difficult to discover this and it isn't clear for us, researchers.
